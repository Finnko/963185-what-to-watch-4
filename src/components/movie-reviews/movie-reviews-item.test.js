import React from 'react';
import renderer from 'react-test-renderer';
import MovieReviewsItem from './movie-reviews-item.jsx';

const props = {
  id: `1`,
  comment: `Good comment!`,
  rating: 8.9,
  user: `Ozzy Osbourne`,
  date: new Date(1583591483969)
};

describe(`MovieReviewsItem component render correctly`, () => {
  it(`Should MovieReviewsItem component render correctly`, () => {
    const tree = renderer
      .create(
          <MovieReviewsItem
            {...props}
          />
      ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
