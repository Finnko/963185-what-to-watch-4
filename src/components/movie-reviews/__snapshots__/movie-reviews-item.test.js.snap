// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieReviewsItem component render correctly Should MovieReviewsItem component render correctly 1`] = `
<div
  className="review"
>
  <blockquote
    className="review__quote"
  >
    <p
      className="review__text"
    >
      Good comment!
    </p>
    <footer
      className="review__details"
    >
      <cite
        className="review__author"
      >
        Ozzy Osbourne
      </cite>
      <time
        className="review__date"
        dateTime="2020-03-07"
      >
        March 07, 2020
      </time>
    </footer>
  </blockquote>
  <div
    className="review__rating"
  >
    8.9
  </div>
</div>
`;
